### DISCUSSION TITLE
How to get same visitor id across all browsers on a same device?

### DISCUSSION THREAD

**Discussion by beqramo** [fingerprintjs/fingerprintjs#896] [Q&A]
Hi,
We have a product where we require users to have one device and one account, and because of that, we need a solution for getting the user's device id.
As I saw with this library demo on a different browser I'm receiving another id so this will not work well.
Is there any configuration that I can do so that it will fit my use case?

Thanks.

**Comment by makma** [comment]
Hello @beqramo,
No, this library performs browser fingerprinting and cannot identify devices.

**Reply by beqramo** [reply]
Thank for your answer, do you know any lib that offers that?
